ideascube/serveradmin/catalog.py

```diff
@@ -253,6 +261,8 @@ def install(self, download_path, install_dir):
 
         for path in glob(os.path.join(datadir, '*', '{}*'.format(zimname))):
             new = path.replace(zimname, '{0.id}.zim'.format(self))
+            if Path(new).exists():
+                rm(new)
             os.rename(path, new)
```

Comment:
I would do it this way instead:

```python
try:
    os.rename(path, new)

except IsADirectoryError:
    rm(new)
    os.rename(path, new)
```

Because it's easier to ask for forgiveness than permission, and because it removes the need to create a `Path` object just to see if the path exists.

Also, it's not a problem if the path exists, it only is one if it's a directory. (if it's a file, it gets written over)